yumljira/templates/Name.vue

```diff
@@ -0,0 +1,16 @@
+<template>
```

Comment:
I have two questions:
Are we using two or four spaces in js?
Do you want to add eslint file?